gk/main.c

```diff
@@ -2367,13 +2368,15 @@ gk_stage1(void *arg)
 	int ret, i;
 	unsigned int num_mbuf;
 
-	gk_conf->instances = rte_calloc(__func__, gk_conf->num_lcores,
-		sizeof(struct gk_instance), 0);
+	gk_conf->instances = rte_calloc_socket(__func__, gk_conf->num_lcores,
+		sizeof(struct gk_instance), 0,
+		rte_lcore_to_socket_id(gk_conf->lcores[0]));
 	if (gk_conf->instances == NULL)
 		goto cleanup;
 
-	gk_conf->queue_id_to_instance = rte_malloc(__func__,
-		num_rx_queues * sizeof(*gk_conf->queue_id_to_instance), 0);
+	gk_conf->queue_id_to_instance = rte_malloc_socket(__func__,
+		num_rx_queues * sizeof(*gk_conf->queue_id_to_instance), 0,
+		rte_lcore_to_socket_id(gk_conf->lcores[0]));
```

Comment:
Avoid duplicating the expression `rte_lcore_to_socket_id(gk_conf->lcores[0])` with a variable.

Reply:
In spite of your comment "This rebase is ready for another review." later in this pull request, my comment above hasn't been addressed.

Reply:
A variable is declared [here](https://github.com/AltraMayor/gatekeeper/pull/383/files#diff-8838f4a2087c3754b8352d7246d9b71dR2370) to avoid duplicating the expression. Are there any more things to do?

Reply:
The confusion here is that you've made that change in the patch **cps: allocate memory on proper numa node** instead in the patch **gk: allocate memory on proper numa node**. Please correct the placement of the code.

Reply:
I see. Sorry for the confusion.

Reply:
The placement of code has been corrected.